tests/test_misc.py

```diff
@@ -169,3 +190,145 @@ async def test_bucket_enforces_encryption(
     #     ServerSideEncryption="aws:kms",
     #     SSEKMSKeyId=authorized_kms_key_arn,
     # )
+
+
+@pytest.mark.asyncio
+async def test_delete_user_bucket(client, access_token_patcher, mock_aws_services):
+    """
+    The user should be able to delete their own bucket.
+    """
+
+    # Create the bucket if it doesn't exist
+    res = await client.get("/storage/info", headers={"Authorization": "bearer 123"})
+    bucket_name = res.json().get("bucket")
```

Comment:
```suggestion
    bucket_name = res.json()["bucket"]
```
it should return it. we don't want to get a `None` value and have to debug why the `head_bucket` call returns an error